styx-flyte-client/src/main/java/com/spotify/styx/flyte/client/FlyteAdminClient.java

```diff
@@ -56,7 +59,7 @@ public static FlyteAdminClient create(String target, boolean insecure) {
     }
     // Enable transparent retries:
     // https://github.com/grpc/proposal/blob/master/A6-client-retries.md#transparent-retries
-    var channel = builder.enableRetry().maxRetryAttempts(0).build();
+    var channel = builder.enableRetry().maxRetryAttempts(MAX_RETRY_ATTEMPTS).build();
```

Comment:
not entirely sure how this client retry logic would work on errors from network.